src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -115,8 +115,8 @@ def __setup_cluster_and_execute_test_config(self, config):
             )
 
     def __execute_integtest_sh(self, endpoint, port, security, test_config):
-        script = self.script_finder.find_integ_test_script(
-            self.component.name, self.repo.dir
+        script = ScriptFinder.find_integ_test_script(
+            self.component.name, self.repo.working_directory
```

Comment:
There are two parts:

1. Where is the integtest.sh to run.
2. Where to run it.

PR #529 addresses (2), but keeps integtest.sh in the root. That means that for hybrid plugins the script itself needs to know whether it's run for, for example, a gradle plugin vs. a dashboards plugin. This is exactly the problem described in https://github.com/opensearch-project/opensearch-build/issues/632 and addressed by this PR. Instead of having to do the `if` inside the script, create two scripts in the gradle or dashboards subdirectory.

Reply:
Ah got it! So if such a plugin wants to add custom script, it should be in gradle or dashboards subdirectories and not on the root. Makes sense, thank you!